detection_rules/docs.py

```diff
@@ -431,6 +470,7 @@ def add_content_to_table_top(self, file_path: Path, new_content: str):
     def generate(self) -> Path:
         self.generate_appendix()
         self.generate_summary()
+        self.generate_rule_reference()
```

Comment:
I like the flow of this `generate` function. Makes it more flexible if we need to add something in as you just did.